src/components/layout/styles.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const LayoutContainer = styled.article`
+  height: 100%;
+  width: 100%;
+`;
+
+export const LayoutWrapper = styled.div`
+  min-height: calc(100% - 200px);
```

Comment:
It would be better to have the header and footer height inside theme as constants. Then you could use them here to calculate the height. The benefit would be that if the designer decides to change the size of footer or header you could easily change it.

But I'll leave it up to you and approve anyways. Feel free to either resolve the conversation or send another commit

Reply:
added to theme

Reply:
Perfect!